app/models/manageiq/providers/autosde/inventory/collector/storage_manager.rb

```diff
@@ -30,13 +30,26 @@ def storage_hosts
     @storage_hosts ||= @manager.autosde_client.StorageHostApi.storage_hosts_get
   end
 
-  def volume_mappings
-    @volume_mappings ||= @manager.autosde_client.StorageHostVolumeMappingApi.storage_hosts_mapping_get.map do |mapping|
+  def host_volume_mappings
+    @host_volume_mappings ||= @manager.autosde_client.StorageHostVolumeMappingApi.storage_hosts_mapping_get.map do
+    |mapping|
```

Comment:
Not sure if you have an automated code formatter, but putting the `|mapping|` on its own line (esp on the same column as the line above) is not great.

We don't have hard-and-fast line limits for this reason, it can make the code _more_ unreadable.

Recommend just putting this on the same line as the `do`

Reply:
done